### ISSUE TITLE
ESLint output cannot be captured

### ISSUE THREAD

**Issue by nkbt** [aghassemi/tap-xunit#5]
tap-xunit expects particular TAP format, which is slightly different for ESLint

ESLint output

```sh
eslint . --format tap
20:55 $ ./node_modules/.bin/eslint . --format tap
TAP version 13
1..12
ok 1 - /Users/nkbt/github/npm-http-server/modules/__tests__/getProperty-test.js
ok 2 - /Users/nkbt/github/npm-http-server/modules/__tests__/parsePackageURL-test.js
ok 3 - /Users/nkbt/github/npm-http-server/modules/createPackageURL.js
ok 4 - /Users/nkbt/github/npm-http-server/modules/getMaxAge.js
ok 5 - /Users/nkbt/github/npm-http-server/modules/getPackage.js
ok 6 - /Users/nkbt/github/npm-http-server/modules/getPackageInfo.js
ok 7 - /Users/nkbt/github/npm-http-server/modules/getProperty.js
ok 8 - /Users/nkbt/github/npm-http-server/modules/parsePackageURL.js
not ok 9 - /Users/nkbt/github/npm-http-server/modules/resolveFile.js
  ---
  message: Unnecessary semicolon.
  severity: error
  data:
    line: 24
    column: 2
    ruleId: no-extra-semi
  ...
ok 10 - /Users/nkbt/github/npm-http-server/modules/ResponseUtils.js
ok 11 - /Users/nkbt/github/npm-http-server/modules/serveNPMPackageFile.js
ok 12 - /Users/nkbt/github/npm-http-server/server.js
```

Produced XML

```sh
eslint . --format tap | tap-xunit

<?xml version="1.0"?>
<testsuites>
  <testsuite name="Default" tests="1" failures="1" errors="0">
    <testcase name="#1 undefined">
      <failure>
          ---
          ...
      </failure>
    </testcase>
  </testsuite>
</testsuites>
```

And report in CircleCI (or any other CI), looks quite unhelpful

<img width="942" alt="20151222-210220" src="https://cloud.githubusercontent.com/assets/175264/11952797/637c691e-a8ef-11e5-905b-39211bb909f8.png">

**Comment by aghassemi** [comment]
Was this working for you before and broke with 1.3.0 (which uses a new version of TAP parser) or was never working?

**Comment by nkbt** [comment]
I think I started using it nor a while ago, so maybe did not even catch 1.2

**Comment by aghassemi** [comment]
Using the latest version (1.3.0) I get the following when piping your output: 

```
<?xml version="1.0"?>
<testsuites>
  <testsuite name="Default">
    <testcase name="#1 /Users/nkbt/github/npm-http-server/modules/__tests__/getProperty-test.js"/>
    <testcase name="#2 /Users/nkbt/github/npm-http-server/modules/__tests__/parsePackageURL-test.js"/>
    <testcase name="#3 /Users/nkbt/github/npm-http-server/modules/createPackageURL.js"/>
    <testcase name="#4 /Users/nkbt/github/npm-http-server/modules/getMaxAge.js"/>
    <testcase name="#5 /Users/nkbt/github/npm-http-server/modules/getPackage.js"/>
    <testcase name="#6 /Users/nkbt/github/npm-http-server/modules/getPackageInfo.js"/>
    <testcase name="#7 /Users/nkbt/github/npm-http-server/modules/getProperty.js"/>
    <testcase name="#8 /Users/nkbt/github/npm-http-server/modules/parsePackageURL.js"/>
    <testcase name="#9 /Users/nkbt/github/npm-http-server/modules/resolveFile.js">
      <failure>
          ---
          ...
      </failure>
    </testcase>
    <testcase name="#10 /Users/nkbt/github/npm-http-server/modules/ResponseUtils.js"/>
    <testcase name="#11 /Users/nkbt/github/npm-http-server/modules/serveNPMPackageFile.js"/>
    <testcase name="#12 /Users/nkbt/github/npm-http-server/server.js"/>
  </testsuite>
</testsuites>
```

Could you try 1.3.0 and let me know?

The failure message is missing because tap parser we use expects a certain format to the failure message. I have to look into whether the raw failure text is provided by the parser or not.

**Comment by nkbt** [comment]
Well, I actually use 1.3 =).

Tap output is in the first message.

Yes you do not capture messages from ESLint output, I checked the source-code before submitting issue. I could fix it by myself and PR, but don't have enough time now, work on too many projects =)

**Comment by aghassemi** [comment]
Sorry, is the bug you are referring to the lack of the "Failure message" or the fact that the generated XML you posted above does not match the TAP you posted? (TAP has 12 tests, generated XML has 1). I can't produce the latter, it seems to generate correct XML for me except for the "Failure message" part which I know is a bug.

**Comment by nkbt** [comment]
I am only about error messages for now, not success ones.

Did not know it is known, so raised an issue.

**Comment by aghassemi** [comment]
I see. ok I can look into that. I thought the whole XML was wrong since the one posted does not match the posted TAP at all.

**Comment by nkbt** [comment]
Yeah. That is true too. XML does not include passed checks. But I reckon it
should not since eslint outputs no "tests" and only headers.

**Comment by aghassemi** [comment]
Alright, should be fixed by https://github.com/aghassemi/tap-xunit/commit/93f39811c36fca42ff3561a6a301ef6971e11ca5 . Published on npm as 1.3.1

**Comment by nkbt** [comment]
Awesome \o/ @aghassemi thanks a lot!
